streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/core/graph/executiongraph/ExecutionGraph.java

```diff
@@ -67,6 +98,58 @@ public void setExecutionJobVertexMap(Map<Integer, ExecutionJobVertex> executionJ
     this.executionJobVertexMap = executionJobVertexMap;
   }
 
+
+  /**
+   * generate relation mappings between actors, execution vertices and channels
+   * this method must be called after worker actor is set.
+   */
+  public void generateActorMappings() {
+    LOG.info("Setup queue actors relation.");
+
+    channelGroupedActors = new HashMap<>();
+    actorIdExecutionVertexMap = new HashMap<>();
+
+    getAllExecutionVertices().forEach(curVertex -> {
+
+      // current
+      actorIdExecutionVertexMap.put(curVertex.getActorId(), curVertex);
+
+      // input
+      List<ExecutionEdge> inputEdges = curVertex.getInputEdges();
+      inputEdges.forEach(inputEdge -> {
+        ExecutionVertex inputVertex = inputEdge.getSourceExecutionVertex();
+        String channelId = curVertex.getChannelIdByPeerVertex(inputVertex);
+        addActorToChannelGroupedActors(channelGroupedActors, channelId,
+          inputVertex.getWorkerActor());
+      });
+
+      // output
+      List<ExecutionEdge> outputEdges = curVertex.getOutputEdges();
+      outputEdges.forEach(outputEdge -> {
+        ExecutionVertex outputVertex = outputEdge.getTargetExecutionVertex();
+        String channelId = curVertex.getChannelIdByPeerVertex(outputVertex);
+        addActorToChannelGroupedActors(channelGroupedActors, channelId,
+          outputVertex.getWorkerActor());
+      });
+    });
+
+    LOG.info("Channel grouped actors is: {}.", channelGroupedActors);
```

Comment:
It's not good solution to log map structure of actors.

Reply:
removed